pkg/ocm/clusters.go

```diff
@@ -239,6 +256,57 @@ func (c *Client) GetCluster(clusterKey string, creator *aws.Creator) (*cmv1.Clus
 	}
 }
 
+func (c *Client) getSubscription(clusterKey string) (*amv1.Subscription, bool, error) {
```

Comment:
I would call it `getSubscriptionByExternalID` to avoid confusion

Reply:
And change parameter to `externalID`

Reply:
Changed.